src/views/blocks/CarouselBlock.php

```diff
@@ -1,32 +1,70 @@
 <?php
 /* @var $this \luya\cms\base\PhpBlockView */
-
-if (!$this->isPrevEqual) {
-    $this->appView->registerJs("$('.carousel').carousel()");
+$config = '';
```

Comment:
We should built the configuration of the carousel() in the block and generate the json object with \luya\helpers\Json::encode()